lib/active_merchant/billing/gateways/place_to_pay.rb

```diff
@@ -0,0 +1,196 @@
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class PlaceToPayGateway < Gateway
+      self.test_url = 'https://api-co-dev.placetopay.ws/'
+      self.live_url = 'https://api-co.placetopay.ws/'
+
+      self.default_currency = 'COP'
+
+      self.supported_countries = ['COL']
+      self.supported_cardtypes = %i[visa master american_express discover]
+      self.homepage_url = 'https://www.placetopay.com/'
+      self.display_name = 'PlaceToPay'
+
+      STANDARD_ERROR_CODE_MAPPING = {}
+
+      ISO_FORMAT_DATE = '%Y-%m-%dT%H:%M:%S%:z'
+
+      def initialize(options = {})      
+        requires!(options, :login, :secret_key)
+        @login, @secret_key = options.values_at(:login, :secret_key)
+        super
+      end
+
+      def purchase(money, payment, options = {})
+        post = {}
+        add_auth(post, options)
```

Comment:
Does the gateway only want to implement two transaction types? I see several different transaction types listed in the gateway docs but it's hard to tell if some of them will match up with some of the ActiveMerchant standard transaction types. I see there is a pre-authorization, would this line up with `verify` for instance?

Reply:
This question still hasn't been answered. There are more than two transaction types in the gateway docs and you have several actions that tie in to those transaction types in the `success_from` method below. We can't parse a `success` from these actions if they aren't being utilized in the standard ActiveMerchant transaction type methods